go.mod

```diff
@@ -5,7 +5,7 @@ go 1.17
 require (
 	github.com/elastic/go-elasticsearch/v7 v7.12.0
 	github.com/gorilla/websocket v1.5.0
-	github.com/multiversx/mx-chain-core-go v1.1.30
+	github.com/multiversx/mx-chain-core-go v1.1.36
 	github.com/multiversx/mx-chain-logger-go v1.0.11
 	github.com/multiversx/mx-chain-vm-common-go v1.3.34
```

Comment:
please change this one to `v1.3.39`
https://github.com/multiversx/mx-chain-vm-common-go/releases/tag/v1.3.39

Reply:
changed